entaxy-project/entaxy
# added category field to transactions and csv parsers.

### Summary

added category field to transactions and csv parsers

### Screenshot
<img width="856" alt="screen shot 2019-01-25 at 11 02 30 am" src="https://user-images.githubusercontent.com/6880373/51757203-c7bde080-2090-11e9-9d30-7ecc3459ab4e.png">

<img width="1204" alt="screen shot 2019-01-25 at 11 02 40 am" src="https://user-images.githubusercontent.com/6880373/51757225-d6a49300-2090-11e9-8a39-35a63fba7465.png">

### How to test it

List step-by-step how to test the changes:

- Step One
upload transaction csv or manually create a transaction.
- Step Two
open the `edit` modal and enter in a category
- Step Three
hit save and verify that the category shows up in the table and also in the redux store

### Other Information

If there's anything else that's important and relevant to your pull
request, mention that information here. This could include
benchmarks, or other information.

Ended up commiting using --no-verify because it was not meeting the test coverage threshold. I suspect that this was the case before my commit too, but didn't verify (sorry). @JackNeto let me know if I am wrong, happy to create snapshot tests

Thanks for contributing to Entaxy!
